### PR TITLE
unmodifiableMap vs Map.of

### PR DESCRIPTION
**Pull request by LucasBulczak**
The provided solution discusses the differences between the two methods above.
I would be grateful for your comments.

### CODE REVIEW

File: src/main/java/com/epam/prejap/ess/helpers/get/map/CreateAMap.java [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+ * (ctrl + shift + T on {@link CreateAMap}).</p>
+ *
+ * @author Łukasz Bulczak
+ * @see Collections
+ * @see Map
+ * @see <a href="http://openjdk.java.net/jeps/269">JEP 269: Convenience Factory
+ * Methods for Collections</a>
+ * @since 0.4
+ */
+class CreateAMap {
```
Comment (marcin-ogorzalek-EPAM) [inline src/main/java/com/epam/prejap/ess/helpers/get/map/CreateAMap.java:54]: In class's name we rather don't use articles. `CreatMap` is enough.

File: src/test/java/com/epam/prejap/ess/helpers/get/map/CreateAMapTest.java [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+    private final BiFunction<String, String, String> mergingFunction =
+            (value1, value2) -> value1.equals(value2)
+                    ? value1
+                    : value1 + ", " + value2;
+
+    private SoftAssertions softly;
+
+    @BeforeMethod
+    void setUp() {
+        softly = new SoftAssertions();
```
Comment (marcin-ogorzalek-EPAM) [inline src/test/java/com/epam/prejap/ess/helpers/get/map/CreateAMapTest.java:28]: An intersting API form `AssertJ`

File: src/main/java/com/epam/prejap/ess/helpers/get/map/CreateAMap.java [hunk 3]
```diff
@@ -0,0 +1,85 @@
+package com.epam.prejap.ess.helpers.get.map;
```
Comment (marcin-ogorzalek-EPAM) [inline src/main/java/com/epam/prejap/ess/helpers/get/map/CreateAMap.java:1]: Why do we have `.get.` in packages?

File: src/main/java/com/epam/prejap/ess/helpers/get/map/CreateAMap.java (hunk 1 above)
Comment (LucasBulczak) [reply src/main/java/com/epam/prejap/ess/helpers/get/map/CreateAMap.java:54]: Good point, thx!

**Comment by LucasBulczak** [comment]
@marcin-ogorzalek-EPAM, thx! Ready for the next round ;-)

File: src/test/java/com/epam/prejap/ess/helpers/get/map/CreateAMapTest.java (hunk 2 above)
Comment (LucasBulczak) [reply src/test/java/com/epam/prejap/ess/helpers/get/map/CreateAMapTest.java]: at first I had everything in separate tests, but it was very illegible in the context of what I wanted to show -> thet any operation causing the modification throws UOEx :-)

AssertJ is a very interesting thing ;-)

**Comment by marcin-ogorzalek-EPAM** [comment]
Bump up the version and your tests to `testng.xml`

**Comment by LucasBulczak** [comment]
Done. Thx for info :-)

**Pull request merged**